pkgs/modules/python/sitecustomize.py

```diff
@@ -25,3 +25,9 @@ def pip_build_shebang(self, executable, post_interp):
 if exe.endswith('/pip') or exe.endswith('/pip3') or exe.endswith('/.pip-wrapped'):
     from pip._vendor.distlib.scripts import ScriptMaker
     ScriptMaker._build_shebang = pip_build_shebang
+
+import os
+pythonpath = os.getenv('REPLIT_PYTHONPATH')
+if pythonpath:
+  paths = pythonpath.split(':')
+  sys.path.extend(paths)
```

Comment:
This would prevent users from ever overriding anything that's in the standard path, we should instead
```suggestion
  paths.extend(sys.path)
  sys.path = paths
```

Reply:
Not correct!